coinChange.cpp

```diff
@@ -0,0 +1,19 @@
+//time complexity: O(n)
+//space complexity:O(1)
+//approach: brute force
+//solved on leetcode? yes, but only 35/182 test cases pass
+//problem faced? yes, I could not think of any other approach, specially dp approach.
+class Solution {
+public:
```

Comment:
Great work trying out all the approaches. Try to solve the solution using the approach taught today. Let me know if you still have doubts. Submit the updated code